src/test/java/com/coveo/spillway/SpillwayTest.java

```diff
@@ -384,4 +394,74 @@ public void triggersAreIgnoreIfTheStorageReturnsAnIncoherentResponse() {
 
     verify(callback, never()).trigger(any(LimitDefinition.class), any());
   }
+
+  @Test
+  public void canAddCapacityLimitOverride() {
+    LimitOverride override = LimitOverrideBuilder.of(JOHN).to(A_CAPACITY).per(A_DURATION).build();
```

Comment:
I think what @GuiSim was proposing is that instead of having only override for the same property as the limit, we could add an override for another property. Ex:
- User(john, 192.168.0.2), User(john, 10.0.0.3) & User(gina, 127.0.0.2)
- Base limit that throttles per IP at 3 calls/s
- An override for "john" at 6 calls/s